arctic/store/_pandas_ndarray_store.py

```diff
@@ -253,8 +262,8 @@ def from_records(self, recarr):
         name = recarr.dtype.names[-1]
         return Series.from_array(recarr[name], index=index, name=name)
 
-    def can_write(self, version, symbol, data):
-        if isinstance(data, Series):
+    def can_write(self, version, symbol, data, **kwargs):
+        if isinstance(data, Series) and 'chunk_size' not in version:
```

Comment:
I can adjust the ordering then and remove this. The code will then rely on them to be in the correct behavior in order to work correctly.